src/redux/sagas/expenditures/releaseExpenditureStages.ts

```diff
@@ -55,30 +59,74 @@ function* releaseExpenditureStage({
       );
     }
 
-    const [permissionDomainId, childSkillIndex] = yield getPermissionProofs(
-      colonyClient.networkClient,
-      colonyClient,
-      expenditure.nativeDomainId,
-      ColonyRole.Arbitration,
-      stagedExpenditureAddress,
+    const stagedExpenditureClient = yield colonyManager.getClient(
+      ClientType.StagedExpenditureClient,
+      colonyAddress,
     );
 
+    const [extensionPermissionDomainId, extensionChildSkillIndex] =
+      yield getPermissionProofs(
+        colonyClient.networkClient,
+        colonyClient,
+        expenditure.nativeDomainId,
+        ColonyRole.Arbitration,
+        stagedExpenditureAddress,
+      );
+
+    let userPermissionDomainId;
+    let userChildSkillIndex;
+    if (expenditure.ownerAddress !== userAddress) {
+      [userPermissionDomainId, userChildSkillIndex] = yield getPermissionProofs(
+        colonyClient.networkClient,
+        colonyClient,
+        expenditure.nativeDomainId,
+        ColonyRole.Administration,
```

Comment:
I'm guessing this is intended to be `Arbitration` as the function call is `releaseStagedPaymentViaArbitration`? I also tried testing as a user with only `Administration` to be sure and the contract call failed.

Reply:
You're completely right! Should be fixed now